cray-hms-smd.compatibility.yaml

```diff
@@ -1,13 +1,20 @@
 ---
 # CSM Version is not really following semver.
 chartVersionToCSMVersion:
+  # Summary:
+  #   Chart Version: 2.0.0 <= x.y.z < 2.1.0, CSM Version: 1.2.0 <= x.y.z < 1.3.0
+  #   Chart Version: 2.1.0 <= x.y.z < 5.0.0, CSM Version: 1.3.0 <= x.y.z < 1.4.0
+  #   Chart Version: 5.0.0 <= x.y.z < 6.0.0, CSM Version: 1.4.0 <= x.y.z < 1.5.0
+  #   Chart Version: 6.0.0 <= x.y.z < 7.1.0, CSM Version: 1.5.0 <= x.y.z < 1.6.0
+  #   Chart Version: 7.1.0 <= x.y.z,         CSM Version: 1.6.0 <= x.y.z
+  #
   # Chart version: CSM version
-  ">=2.0.0": "~1.2.0" # Chart Version: 2.0.0 <= x.y.z, CSM Version: 1.2.0 <= x.y.z < 1.3.0
+  ">=2.0.0": "~1.2.0"
   ">=2.1.0": "~1.3.0"
   ">=4.0.0": ">=1.3.0"
   ">=5.0.0": ">=1.4.0" # CSM 1.4.0 is locked on chart version 5.0.0 and using app version 2.2.*
                        # If chart changes are needed a 5.0.4 chart with app 2.2.* is likely possible
-  ">=6.0.0": ">=1.5.0" # CSM 1.5.0 is using chart 7.0.* and app version 2.11.*
```

Comment:
I think this comment is still good. At least some from of it that says, app changes have to be to 2.11.* for csm 1.5.0

Reply:
CSM 1.5.x used both helm charts 6.0.x and 7.0.x, which included app versions 2.2.0 through 2.9.0 and 2.11.x.  That got pretty complex to put into a one line comment so I removed it because the new comment block at the top now shows the chart <-> CSM version and the table at the bottom already shows what app version is in each chart version.

Reply:
The comment in this case, is to just comment what app version to use on 1.5
It doesn't need to include the history

Reply:
Fixed, per slack discussion.